server/vulnerabilities/nvd/cve_test.go

```diff
@@ -474,6 +478,9 @@ func TestTranslateCPEToCVE(t *testing.T) {
 			}
 
 			for _, cve := range tc.excludedCVEs {
+				for _, cveFound := range cvesFound[cpe] {
+					require.NotEqual(t, cve, cveFound.ID, fmt.Sprintf("%s should not contain %s", cpe, cve))
+				}
```

Comment:
Check below `require.NotContains(t, cvesFound[cpe], cve, tc.cpe)` should be removed?

Reply:
ah good catch